src/Raven.Server/Documents/DocumentFlags.cs

```diff
@@ -18,5 +18,6 @@ public enum DocumentFlags
         Reserved3 = 0x80,
 
         HasAttachments = 0x100,
+        Revision = 0x200
```

Comment:
Maybe since this is used only in code that handles legacy stuff, we can use DocumentFlags.Versioned ? We would avoid having a new value here + avoid having 'NonPersistentFlags'. Just an idea.